packages/launchpad/cypress/e2e/onboarding-flow.cy.ts

```diff
@@ -33,6 +33,34 @@ describe('Launchpad: Onboarding Flow', () => {
     cy.findByText('Choose a Browser', { timeout: 10000 })
   })
 
+  it('can setup component testing with TS', () => {
+    cy.visitLaunchpad()
+    cy.get('[data-cy-testingType=component]').click()
+    cy.get('[data-testid=select-framework]').click()
+    cy.findByText('React.js').click()
+    cy.get('[data-testid=select-framework]').should('contain', 'React.js')
+    cy.get('[data-testid=select-bundler]')
+    .findByText(cy.i18n.setupPage.projectSetup.bundlerPlaceholder)
+    .click()
+
+    cy.findByText('Webpack').click()
+    cy.get('[data-testid=select-bundler]').should('contain', 'Webpack')
+    cy.reload()
+    cy.get('[data-testid=select-framework]').should('contain', 'React.js')
+    cy.get('[data-testid=select-bundler]').should('contain', 'Webpack')
+    cy.findByText('TypeScript').click()
+    cy.findByText('Next Step').click()
+    cy.get('h1').should('contain', 'Dependencies')
+    cy.findByText('I\'ve installed them').click()
+    cy.findByText('We added the following files to your project.')
+    cy.findByText('Continue').click()
+    cy.withCtx(async (ctx) => {
+      return ctx.file.readFileInProject('cypress.config.ts')
+    }).then((str) => {
+      cy.log(str)
```

Comment:
Is this just for debugging or is this part of the test? Should there be an assertion here?